scripts/isolate_tests.py

```diff
@@ -91,5 +91,8 @@ def write_cases(tests):
             if docs:
               cases = extract_docs_cases(path)
             else:
-              cases = extract_test_cases(path)
+              if f.endswith(".sol"):
+                cases = [open(path, "r").read()]
```

Comment:
This is weird, I would have expected the tests to fail for .sol files that import other .sol files. Can you please check?

Reply:
I'm just looking through the test scripts and tried to look up where isolate_tests.py is actually used.

The first case is cmdlineTests.sh - here the files collected from test/ are only tested using solfuzzer - however, it seems that for me solfuzzer succeeds even for entirely random input, which I suppose shouldn't be the case... still trying to figure out what's happening there.

The second case is storebytecode.sh, which ultimately calls prepare_report.py, which creates a report.txt, in which all tests with imports are reported to fail as excepted. Still trying to figure out why that doesn't result in reported test failures.

So I'm still investigating why this does not cause more problems (which seems rather weird), but ultimately it will probably make sense to exclude test/compilationTests/ in isolate_tests.py.

Reply:
storebytecode.sh seems to only be run on the travis and at the moment those tests fail earlier due to the older boost version.

Reply:
solfuzzer is meant to detect crashes and internal compiler errors inside the compiler. It is meant to be run with a fuzzing library and should fail only in these cases, but the tests also just run it on specific inputs.

storebytecode is used to detect differences in compilation across different platforms. The bytecode is stored in a specific github repository and the files are compared once we have all architectures.

So all of this is fine, the question is whether we actually need it.